visualization/app/codeCharta/ui/customConfigs/customConfigs.component.scss

```diff
@@ -1,9 +1,15 @@
 .custom-configs {
-	margin-left: 15%;
 }
 
 .custom-configs-component {
-	.custom-configs-button {
+	display: inline-block;
+	width: calc(50% - 2.5px);
```

Comment:
do we need this expensive `calc`? I would prefer alignment with flex or to just hard code some pixel width